nifi-nar-bundles/nifi-extension-utils/nifi-prometheus-utils/pom.xml

```diff
@@ -53,6 +53,11 @@
             <artifactId>nifi-metrics</artifactId>
             <version>2.0.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-framework-api</artifactId>
+            <version>2.0.0-SNAPSHOT</version>
+        </dependency>
```

Comment:
We should not have extensions / utils depending on nifi-framework-api. That is solely for use by the nifi framework and can change at any point in time.
I would be okay with moving the `StorageUsage` class to nifi-api and adding methods to `EventAccess` to expose the `StorageUsage` for the particular repositories, but we should not be exposing things like SystemDiagnostics to extensions.

Reply:
Thank @markap14 for your review. I've modified the code based on your recommendations.